deploy/docker/docker-entrypoint.sh

```diff
@@ -1,41 +1,43 @@
-#!/bin/sh
+#!/bin/bash
 ## EMQ docker image start script
 # Huang Rui <vowstar@gmail.com>
 # EMQ X Team <support@emqx.io>
 
 ## Shell setting
-if [[ ! -z "$DEBUG" ]]; then
+if [[ -n "$DEBUG" ]]; then
     set -ex
 else
     set -e
 fi
 
+shopt -s nullglob
```

Comment:
For globs loop.
It's used to enumerate plugins config, related https://github.com/koalaman/shellcheck/wiki/SC2045